docs/intro.rst

```diff
@@ -292,10 +292,35 @@ FromBytes
 Bool
 ....
 
+The checker test that a received value is a boolean type.
+
+    >>> t.Bool().check(True)
+    True
 
 StrBool
 .......
 
+If you need to check value that can be equivalent to a boolean type, you can use ``StrBool``.
+**Letter case doesn't matter.**
+
+As an example, ``StrBool`` can be used to classify data with dissimilar values.
+
+    >>> data = {'Alice': 'YeS', 'Bobby': '1', 'Johny': 0, \
+    >>>   'Linda': 't', 'Tomas': 'oFF', 'Aaron': None, \
+    >>>   'Isaac': 'On', 'Carry': False, 'Terry': 1}
+    >>>
+    >>> for key, value in data.items():
+          print("'%s' is %s" % (key, t.StrBool().check(value)))
+    'Alice' is True
+    'Bobby' is True
+    'Johny' is False
+    'Linda' is True
+    'Tomas' is False
+    'Aaron' is False
+    'Isaac' is True
+    'Carry' is False
+    'Terry' is True
```

Comment:
In my opinion you can make better example. What if we just show when false is False and True is true?

Some like that.

```
for value in ['YeS', 1, 't', ...]:
    print(i, t.StrBool().check(value))
```

This is so similar with your example, but now u can see that `t -> True`.

What do u think?

Reply:
This was my first example, but I decided to replace it with something more lively. 
Moreover, we have something similar here: https://trafaret.readthedocs.io/en/latest/api/trafaret.html#trafaret.StrBool.

Also, all equivalents will be described above and I don't quite understand the value of this example.

**Here is my previous example:**
Equivalents for `True`:
```python
true_values = ('t', 'true', 'y', 'yes', 'on', '1')

for true in true_values:
    print("%s is %s" % (true, t.StrBool().check(true)))
```
```
t is True
true is True
y is True
yes is True
on is True
1 is True
```

Equivalents for `False`:
```python
false_values = ('false', 'n', 'no', 'off', '0', 'none')

for false in false_values:
    print("%s is %s" % (false, t.StrBool().check(false)))
```

```
false is False
n is False
no is False
off is False
0 is False
none is False
```

Reply:
we can not describe `equivalents` and show these in our example. Yes, I think we need to replace.